distribution/changelog.txt

```diff
@@ -5,6 +5,7 @@
 - Feature: [#13000] objective_options command for console.
 - Feature: [#13096] Add Esperanto translation.
 - Change: [#9568] Change lift sounds of Reverser Roller Coaster and Compact Inverted Coaster to better fitting ones.
+- Fix: [#13102] Correct underflow when displaying negative length.
```

Comment:
We keep the changelog ordered in numerical order, grouped by category. Could you move this line down, so the order is retained?

Reply:
My bad, will fix.